update-trads.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
should `set -eo pipefail` so the script exits if it fails to run git command, download form crowdin or cd for any reason